src/workflows-app/components/TroubleshootingBox.ts

```diff
@@ -0,0 +1,99 @@
+import { div, h, span } from 'react-hyperscript-helpers';
+import { ClipboardButton } from 'src/components/ClipboardButton';
+import { Link } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import { LogTooltips } from 'src/workflows-app/utils/task-log-utils';
+
+type TroubleshootingBoxProps = {
+  name: string;
+  namespace: string;
+  logUri: string;
+  submissionId: string;
+  workflowId: string;
+  showLogModal: (modalTitle: string, logsArray: any, tesLog: string) => void;
+  executionDirectory: string;
+};
+
+export const TroubleshootingBox: React.FC<TroubleshootingBoxProps> = (props) => {
+  const makePath = () => {
+    const splitString = 'cbas/';
+    if (!props.executionDirectory || !props.executionDirectory.includes(splitString)) {
+      return 'workspace-services/cbas/';
+    }
+    const path = props.executionDirectory.split(splitString)[1];
+    return `workspace-services/cbas/${path}/`;
+  };
+
+  return div(
+    {
+      style: {
+        border: `1px solid ${colors.dark(0.4)}`,
+        borderRadius: 5,
+        paddingTop: '0.25em',
+        paddingBottom: '0.25em',
+        paddingLeft: '1em',
+        paddingRight: '1em',
+        lineHeight: '24px',
+        alignSelf: 'flex-start',
+        maxHeight: 'fit-content',
+        margin: '0 10px',
+      },
+    },
+    [
```

Comment:
This might be a good place for a `dl` element.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl